.github/runs-on.yml

```diff
@@ -1,10 +1,14 @@
 runners:
   self-hosted-ubuntu-22.04-x86-64:
-    cpu: [16, 32, 64]
-    ram: [64, 128, 192]
+    # A range from lowest to highest
+    cpu: [16, 64]
+    ram: [64, 192]
     disk: default
-    family: ["c7a", "c7i", "m7a", "m7i"]
+    family: ["c7", "m7", "r7"]
```

Comment:
The reason I had targeted the instance type families is so we don't run on ARM machines. Only the Intel/AMD ones. I don't see any benefit in this but i could be missing something.
https://aws.amazon.com/ec2/instance-types/

Reply:
5th and 6th generation are older processors, better to use the 7 generation processors.